src/processor/InstanceProcessor.ts

```diff
@@ -49,3 +90,6 @@ const CONFORMANCE_AND_TERMINOLOGY_RESOURCES = new Set([
   'NamingSystem',
   'TerminologyCapabilities'
 ]);
+
+// The properties that are handled via keywords
+const IGNORED_PROPERTIES = ['resourceType', 'id', 'meta'];
```

Comment:
Not sure we can totally ignore `meta`. There are several fields on `meta` not handled by any keywords.

Reply:
You're definitely right. I removed `meta` from the list of ignored properties. If the first profile is used as the `InstanceOf` keyword, I do remove it since we won't need to set an Assignment Rule for that. If that then leaves `profile` or `meta` empty, I also remove those properties. Fixed in 8736f5b.